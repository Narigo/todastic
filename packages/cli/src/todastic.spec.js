const todastic = require("./todastic");

describe("command line interface", () => {
  it("can load a file and output information", async () => {
    await expect(todastic(`${__dirname}/__tests__/sample.todo`).tracked()).resolves.toEqual("03:30:00");
  });
});
