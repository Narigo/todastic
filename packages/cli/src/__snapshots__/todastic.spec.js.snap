// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`command line interface can filter by a specific status 1`] = `"02:10:00"`;

exports[`command line interface returns an empty result when filtering for an unused status 1`] = `"00:00:00"`;
